src/components/ContentHighlights/data/hooks.js

```diff
@@ -30,12 +33,41 @@ export function useHighlightSetsForCuration(enterpriseCuration) {
   return highlightSets;
 }
 
+export function useHighlightSet(highlightSetUUID) {
+  const [highlightSet, setHighlightSetItems] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  const getHighlightSetItems = useCallback(async () => {
```

Comment:
nit: `getHighlightSet` since you're not just returning the highlight content associated with the `HighlightSet`, but rather all metadata about the `HighlightSet`